agent/containers/images/Makefile

```diff
@@ -102,11 +106,11 @@ tm: all-tags $(_DEFAULT_DISTROS:%=%-tool-meister-tagged)
 #-
 
 # We also offer targets per distribution target
-${_DISTROS}: %: %-all-tagged
+${_DISTROS}: %: pkgmgr-clean %-all-tagged
 
-$(_DISTROS:%=%-tds): %-tds: %-tool-data-sink-tagged
+$(_DISTROS:%=%-tds): %-tds: pkgmgr-clean %-tool-data-sink-tagged
```

Comment:
Shall we add
```
${_DISTROS:%=%-everything}: %-everything: pkgmgr-clean %-all-tagged
```
to enable (e.g.) `centos-8-everything` as a target (analogous to the `centos-8-tds` target, as the `everything` target is to the `tds` target)?

And, if you want to be DRYer after adding that line, change line 109 to:
```
${_DISTROS}: %: %-everything
```